### ISSUE TITLE
GCC 9.3.0 R2 build - Concerning SEH

### ISSUE THREAD

**Issue by gammaray10k** [brechtsanders/winlibs_mingw#5]
Your build states that it was built with dwarf enabled even though there is a libgcc_s_seh-1.dll in the \bin directory. I wanted to know if you can create a GCC 9.3.0 with SEH enabled, like how niXman's build (the gcc build used by CodeBlocks v20.03) has it enabled.
[gcc_build_comparison.txt](https://github.com/brechtsanders/winlibs_mingw/files/4406126/gcc_build_comparison.txt)

**Comment by brechtsanders** [comment]
Yes, I'm aware of this: 32-bit is dwarf , 64-bit SEH, but both mention dwarf in the name.
Will be fixed in the next build.

**Comment by brechtsanders** [comment]
The just released build ([GCC 9.3.0 + Clang/LLVM 10.0.0 + MinGW-w64 7.0.0 (release 3)](https://github.com/brechtsanders/winlibs_mingw/releases/tag/9.3.0-10.0.0-7.0.0-r3)) fixes this.

32-bit:
```
$ gcc -v
Using built-in specs.
COLLECT_GCC=D:\Prog\winlibs32-9.3.0\mingw32\bin\gcc.exe
COLLECT_LTO_WRAPPER=d:/prog/winlibs32-9.3.0/mingw32/bin/../libexec/gcc/i686-w64-mingw32/9.3.0/lto-wrapper.exe
Target: i686-w64-mingw32
Configured with: ./configure --prefix=/R/winlibs32-9.3.0/inst_gcc-9.3.0/share/gcc --build=i686-w64-mingw32 --host=i686-w64-mingw32 --with-pkgversion='MinGW-W64
i686-posix-dwarf, built by Brecht Sanders' --with-tune=generic --enable-checking=release --enable-threads=posix --with-dwarf2 --disable-sjlj-exceptions --disable-libunwind-exceptions --enable-serial-configure --disable-bootstrap --enable-host-shared --enable-plugin --enable-default-ssp --disable-rpath --disable-libstdcxx-pch --enable-libstdcxx-time=yes --disable-libstdcxx-debug --disable-version-specific-runtime-libs --with-stabs --disable-symvers --enable-languages=c,c++,fortran,lto,objc,obj-c++,d --disable-gold --disable-nls --disable-stage1-checking --disable-win32-registry --disable-multilib --enable-ld --enable-libquadmath --enable-libada --enable-libssp --enable-libstdcxx --enable-lto --enable-fully-dynamic-string --enable-libgomp --enable-graphite --enable-mingw-wildcard --with-mpc=/d/Prog/winlibs32-9.3.0/custombuilt --with-mpfr=/d/Prog/winlibs32-9.3.0/custombuilt --with-gmp=/d/Prog/winlibs32-9.3.0/custombuilt --with-isl=/d/Prog/winlibs32-9.3.0/custombuilt --enable-install-libiberty --enable-__cxa_atexit --without-included-gettext --with-diagnostics-color=auto --with-libiconv --with-system-zlib --with-build-sysroot=/R/winlibs32-9.3.0/gcc-9.3.0/mingw-w64 --enable-large-address-aware
Thread model: posix
gcc version 9.3.0 (MinGW-W64 i686-posix-dwarf, built by Brecht Sanders)
```
64-bit:
```
$ gcc -v
Using built-in specs.
COLLECT_GCC=D:\Prog\winlibs64-9.3.0\mingw64\bin\gcc.exe
COLLECT_LTO_WRAPPER=d:/prog/winlibs64-9.3.0/mingw64/bin/../libexec/gcc/x86_64-w64-mingw32/9.3.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ./configure --prefix=/R/winlibs64-9.3.0/inst_gcc-9.3.0/share/gcc --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 --with-pkgversion='MinGW-W64 x86_64-posix-seh, built by Brecht Sanders' --with-tune=generic --enable-checking=release --enable-threads=posix --disable-sjlj-exceptions --disable-libunwind-exceptions --enable-serial-configure --disable-bootstrap --enable-host-shared
--enable-plugin --enable-default-ssp --disable-rpath --disable-libstdcxx-pch --enable-libstdcxx-time=yes --disable-libstdcxx-debug --disable-version-specific-runtime-libs --with-stabs --disable-symvers --enable-languages=c,c++,fortran,lto,objc,obj-c++,d --disable-gold --disable-nls --disable-stage1-checking --disable-win32-registry --disable-multilib --enable-ld --enable-libquadmath --enable-libada --enable-libssp --enable-libstdcxx --enable-lto --enable-fully-dynamic-string
--enable-libgomp --enable-graphite --enable-mingw-wildcard --with-mpc=/d/Prog/winlibs64-9.3.0/custombuilt --with-mpfr=/d/Prog/winlibs64-9.3.0/custombuilt --with-gmp=/d/Prog/winlibs64-9.3.0/custombuilt --with-isl=/d/Prog/winlibs64-9.3.0/custombuilt --enable-install-libiberty --enable-__cxa_atexit --without-included-gettext --with-diagnostics-color=auto --with-libiconv --with-system-zlib --with-build-sysroot=/R/winlibs64-9.3.0/gcc-9.3.0/mingw-w64
Thread model: posix
gcc version 9.3.0 (MinGW-W64 x86_64-posix-seh, built by Brecht Sanders)
```

This is also the first release that also comes with Clang/LLVM.
